### DISCUSSION TITLE
Access Pybricks Primehub in REPL

### DISCUSSION THREAD

**Discussion by Vinz1911** [pybricks/support#429] [Q&A] [answered]
Hello 👋 

I just tried a few things and want to know if it's possible to access the Primehub with Pybricks Firmware via repl ?
I tried it like with the original firmware but it does not work. Is it possible or (currently) not ?

Thank you and best regards

**Comment by dlech** [comment, marked as answer]
Right now, it only works with BLE, which means it only works with https://code.pybricks.com. There is an open [issue](https://github.com/pybricks/pybricksdev/issues/17) to make it work with `pybricksdev`.

USB is currently disabled.

**Reply by Vinz1911** [reply]
ah okay, thank you :)
